.travis.yml

```diff
@@ -1,16 +1,25 @@
 language: go
 go_import_path: github.com/csi-driver/csi-driver-smb
 dist: bionic
-
+services:
+  - docker
 go:
   - "1.13.9"
 
 before_install:
   - GO111MODULE=off go get github.com/mattn/goveralls
+  - mkdir /tmp/share
+  - chmod 777 /tmp/share
+  - docker run -it --name samba  -p 445:445 -v /tmp/share:/mount -d dperson/samba -u "username;test"  -s "Downloads;/mount/;yes;no;no;all;user"
```

Comment:
shall we use this official samber image: https://hub.docker.com/r/servercontainers/samba ?

Reply:
OK, I will do a test later